src/main/java/uk/gov/ons/ctp/response/action/endpoint/ActionEndpoint.java

```diff
@@ -146,14 +147,23 @@ public ActionDTO findActionByActionId(@PathVariable("actionid") final UUID actio
       throws CTPException, InvalidRequestException {
     log.debug("Entering createAdhocAction with actionPostRequestDTO {}", actionPostRequestDTO);
     if (bindingResult.hasErrors()) {
-      throw new InvalidRequestException("Binding errors for create action: ", bindingResult);
+      throw new InvalidRequestException(
+          "Binding errors for create acti"
+              + "action.setActionPlanFK(actionPlan.getActionPlanPK());on: ",
```

Comment:
Looks like an accidental typo/concatenation on the error message

Reply:
Fixed